### PR TITLE
November Release Notes - Beta Target Directory

### PR DESCRIPTION
**Pull request by stef-cobaltio**
New release note for November 2024 covering the Beta release of target directory.

## Preview This Change

To see how this change looks in production, scroll down to **Deploy Preview**. Select the link that looks like `https://deploy-preview-<num>--cobalt-docs.netlify.app/`

## Variables

Help us support a “Write once, publish everywhere” single source of truth. If you see a line that looks like:

`{{% asset-categories %}}`

You’ve found a [shortcode](https://gohugo.io/content-management/shortcodes/) that we include in multiple documents.

You’ll find the content of the shortcode in the following directory:

https://github.com/cobalthq/cobalt-product-public-docs/tree/main/layouts/shortcodes

That shortcode has the same base name as what you see in the PR, such as `asset-categories.html`.

## Checklist for PR Author

[ ] Did you check for broken links and alt text?

Be sure to check for broken links and Alt Text issues. We have a partially automated process,
as described in this section of our repository README:
[Test Links and Alt Attributes](https://github.com/cobalthq/cobalt-product-public-docs/blob/main/README.md#test-links-and-alt-attributes).

### CODE REVIEW

**Comment by mtsuji-cobalt** [comment]
Screenshot 2024-11-22 at 10 02 30

The Beta label is causing some weird HAHAHUGO to appear on the right-hand side bar. i searched for it in the code but it might be a BE thing? tagging @TaylorBriggs if he can help

File: content/en/Product Updates/release-notes-november-2024.md [hunk 1]
```diff
@@ -9,6 +9,21 @@ description: >-
 Explore What's New from Cobalt This Month
 {{% /pageinfo %}}
 
+
+## Target Directory {{< beta-label >}}
+
+We've introduced a central directory feature that allows customers to conveniently track their targets across Pentests and ASM modules, providing a cohesive and organized approach to managing these crucial units.
```
Comment (mtsuji-cobalt) [inline content/en/Product Updates/release-notes-november-2024.md:15]: I believe it also includes DAST targets

**Comment by stef-cobaltio** [comment]
Yeah, good catch on the beta tag, I moved it so that it wouldn't be a blocker and also updated the copy to include DAST.

**Review by mtsuji-cobalt — APPROVED** [review]

**Comment by TaylorBriggs** [comment]
> Screenshot 2024-11-22 at 10 02 30
> The Beta label is causing some weird HAHAHUGO to appear on the right-hand side bar. i searched for it in the code but it might be a BE thing? tagging @TaylorBriggs if he can help

I think it would need to be like `{{< beta-label />}}` but I didn't try it out myself. It looks like it was trying to wrap some content instead of self-closing.

**Pull request merged**